docs/how-to-guides/faq.md

```diff
@@ -82,15 +82,15 @@ Currently, promptflow supports the following environment variables:
 
 **PF_WORKER_COUNT** 
 
-Valid for batch run only. The number of workers to use for parallel execution of the Flow.
+Effective for batch run only, count of parallel workers in batch run execution.
 
-Default value is 16. If you have large number of batch run date row count, and want more efficiency, you can increase the PF_WORKER_COUNT to improve the batch run concurrency, make it run faster.
+The default value has been updated from 16 to 4.
```

Comment:
As a document, no need to say "updated from 16 to 4"
Maybe `The default value is 4 (was 16 when promptflow<1.4.0)`